themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties

```
@@ -86,30 +134,69 @@ inResource=in
 fullAccess=Volledige toegang
 offlineToken=Offline Token
 revoke=Vergunning intrekken
+
 configureAuthenticators=Ingestelde authenticators
 mobile=Mobiel nummer
 totpStep1=Installeer een van de onderstaande applicaties op uw mobiele apparaat:
 totpStep2=Open de toepassing en scan de QR-code of voer de sleutel in.
 totpStep3=Voer de door de toepassing gegeven eenmalige code in en klik op Opslaan om de configuratie af te ronden.
+totpStep3DeviceName=Geef een apparaatnaam op om je OTP-apparaten te identificeren.
```

Since we're already using the formal 'you' in other places we might as well do so here as well.

```suggestion
totpStep3DeviceName=Geef een apparaatnaam op om uw OTP-apparaten te identificeren.
```